.github/workflows/auto-release.yml

```
@@ -0,0 +1,293 @@
+      always() &&
+      needs.analyze-changes.outputs.should_release == 'true' &&
+      (needs.quality-gate.result == 'success' ||
+       github.event.inputs.skip_tests == 'true')
+
+    permissions:
+      contents: write
+      pull-requests: write
+
+    outputs:
+      new_version: ${{ steps.version.outputs.new_version }}
+      release_created: ${{ steps.create.outputs.release_created }}
```

actionlint (1.7.7)

property "release_created" is not defined in object type {assets: string; id: string; upload_url: string; url: string}

(expression)